virt/virt-architecture.adoc

```diff
@@ -0,0 +1,25 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="virt-architecture"]
+= {VirtProductName} architecture
+:context: virt-architecture
+
+toc::[]
+
```

Comment:
Is "virt-additional-components.adoc" supposed to be included here?

Reply:
Nope. It was in an early draft but that info got mushed into the rest of the modules.